### ISSUE TITLE
Are you interested in becoming tile layer featured on the OSM main site?

### ISSUE THREAD

**Issue by matkoniecz** [cquest/osmfr-cartocss#65]
Someone added this map style to "Proposed layers" in https://wiki.openstreetmap.org/wiki/Featured_tile_layers

AFAIK it was not done on behalf of whoever maintains this map style and its hosting (and therefore ineligible - see https://wiki.openstreetmap.org/wiki/Featured_tile_layers/Guidelines_for_new_tile_layers )

Are you interested in being a new map style featured on the OSM main site? AFAIK this map style fulfills criteria mentioned at https://wiki.openstreetmap.org/wiki/Featured_tile_layers/Guidelines_for_new_tile_layers and it could be interesting expansion of map styles already available

If you are not interested and that addition to wiki was done without consultations - then sorry for bothering you and I will remove this entry from OSM Wiki (I wanted to ask before removing it)

**Comment by Marc-marc-marc** [comment]
imho, support is for the hosting by osm-fr :)
being a layer on osm.org doesn't produce any load on the style itselft

**Comment by matkoniecz** [comment]
> imho, support is for the hosting by osm-fr :)

Yes, that is the main part - but I assumed of there is such plan then maintainers of map style will be aware about it

> being a layer on osm.org doesn't produce any load on the style itselft

As it will be not default and figuring out how to switch layers is a bit tricky it will not be huge.

But bigger exposure will result in more feature requests and more bug reports so there is also some cost here

**Comment by Tigerfell** [comment]
There are similar feature requests, but limited to the wiki only
- #61 
- osm-fr/infrastructure/issues/153

I even opened a PR at https://github.com/openstreetmap/chef/pull/278
I filed those issues in February 2020, but I would like to finish them off now. @matkoniecz can we join our efforts and make a general request for the main page and the wiki map displays?

**Comment by cquest** [comment]
We first need to complete a cleanup step in the style, make it a bit lighter on required resources.
We also need to switch to another expire/refresh strategy (the tile server is currently overloaded).
Another step... switch also to new hardware if possible.

**Comment by Tigerfell** [comment]
As I wrote in https://github.com/osm-fr/infrastructure/issues/153#issuecomment-660690429, I formally proposed the addition of the tile layer on osm.org on 19 July to OWG so you would need to talk to them if you want to withdraw or postpone the proposal. AFAIK, they will also get in touch with OSM-FR, so maybe @cquest's comment was in response to that?

I hoped to finally merge the code in openstreetmap/chef/pull/278 (open since February), which depends on this. I will open a new PR now, which does not depend on this issue, so I can finally fix the broken map displays in the OSM-wiki.

**Comment by cquest** [comment]
So far, there is not official request coming from OSM-FR to be added on osm.org.
We really need to enhance our servers prior to request it.

We did it for CyclOSM, improving the servers, adding a front cache, test a full day of humanitarian tiles requests on cyclosm and check everything was ok. For the "FR" style we're just not ready yet but interested.

**Comment by Tigerfell** [comment]
> 
> 
> So far, there is not official request coming from OSM-FR to be added on osm.org.
> We really need to enhance our servers prior to request it.
> 
> We did it for CyclOSM, improving the servers, adding a front cache, test a full day of humanitarian tiles requests on cyclosm and check everything was ok. For the "FR" style we're just not ready yet but interested.

**I @Tigerfell ** filed a request that OSM-FR should become a featured tile layer. Based on previous comments, I understood that it would be fine for you. Nonetheless, OSM-FR will not be added without the consent of the tile provider...
